src/test/java/baseball/domain/ComputerTest.java

```diff
@@ -0,0 +1,33 @@
+package baseball.domain;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+@DisplayName("Computer Unit Test")
+class ComputerTest {
+
+	@Test
+	@DisplayName("랜덤 숫자를 추출 후 1000회 검증")
+	void extractRandomValue() {
+		for (int i = 0; i < 1000; i++) {
```

Comment:
```suggestion
		for (int i = 0; i < 1_000; i++) {
```
1,000 이상의 수는 이런식으로 가독성을 높일 수도 있답니다. 😉